composer.json

```diff
@@ -10,14 +10,14 @@
         }
     ],
     "require": {
-        "php": "^5.4 || ^7.0",
-        "blablacar/redis-client": "^1.0"
+        "php": "^7.1",
```

Comment:
`^7.1 || ^8.0` or `>= 7.1` ?

Reply:
We should not go for an open condition as we don't know what will be the BC breaks on future versions.
Same on PHP 8. I'm not confident this bundle runs properly on PHP 8.